packages/data-explorer-ui/src/components/Error/error.tsx

```diff
@@ -7,15 +8,46 @@ import { SectionActions } from "../common/Section/section.styles";
 import { PRIORITY, StatusIcon } from "../common/StatusIcon/statusIcon";
 import {
   Error as CustomError,
+  ErrorDetailSectionContent,
+  ErrorDetailsWrapper,
   ErrorSection,
   SectionContent,
 } from "./error.styles";
 
+interface ErrorDetailSectionProps {
+  detail: string;
+  title: string;
+}
+
+const ErrorDetailSection = ({
+  detail,
+  title,
+}: ErrorDetailSectionProps): JSX.Element => (
+  <Grid container spacing={2}>
```

Comment:
And this is my mistake totally, here, we have a styled component called `SectionContent` which you can use (I forgot to mention it in the ticket). You'll have a class in import names so perhaps create an import alias. Remove all Grid containers and Grid items.